packages/network-controller/tests/provider-api-tests/no-block-param.ts

```diff
@@ -77,7 +77,7 @@ export const testsForRpcMethodAssumingNoBlockParam = (
   });
 
   for (const paramIndex of [...Array(numberOfParameters).keys()]) {
-    it(`does not reuse the result of a previous request if parameter at index "${paramIndex}" differs`, async () => {// eslint-disable-line
+    it(`does not reuse the result of a previous request if parameter at index "${paramIndex}" differs`, async () => {
```

Comment:
My editor removed this when I formatted the document. I'm not sure why it was there in the first place, but it appears to have been unnecessary.